CMakeLists.txt

```diff
@@ -73,6 +73,8 @@ fs_test_env(NAME dev.threads_nox_noopengl ENVFILE env/dev/sim_threads_no-x_no-op
 fs_test_env(NAME jun19.threads   ENVFILE env/jun19/sim_threads.yaml)
 fs_test_env(NAME jun19.nothreads ENVFILE env/jun19/sim_no-threads.yaml)
 
+fs_test_env(NAME r3broot ENVFILE env/r3broot/dev.yaml)
```

Comment:
This is introducing a new naming scheme that is conflicting with the one we have at the moment which is `env/<version>/<variant>.yaml`. What about `env/dev/r3broot.yaml`?

Reply:
agree with Dennis

Reply:
Renamed.

Reply:
marking resolved. If that's a problem, please complain.